app/views/shared/_breadcrumbs.html.erb

```diff
@@ -1,7 +1,25 @@
-<div class="container breadcrumbs-container" role="navigation" aria-label="<%= t('spotlight.breadcrumb.label') %>">
-  <div class="row">
-    <div class="col-md-12">
-      <%= render_breadcrumbs builder: Spotlight::BootstrapBreadcrumbsBuilder %>
+<% if defined?(breadcrumbs) && breadcrumbs.present? %>
```

Comment:
This got added because spec/features/confirm_email_spec.rb:8 keeps failing without it.

Reply:
Following this ```

I think if you define these in Spotlight::Controller instead and add breadcrumbs to the list of helpers in the included do block also in Spotlight::Controller you could remove the separate helper_method :breadcrumbs entries in CatalogController and ApplicationController.
```

Fixed the problem. I already had a component already made so I just added it anyway.